src/operations/command.ts

```diff
@@ -55,6 +55,7 @@ export interface CommandOperationOptions
   // Admin command overrides.
   dbName?: string;
   authdb?: string;
+  /** @deprecated Will be removed in the next major version. Set writeConcern.w to 0 instead. */
   noResponse?: boolean;
```

Comment:
More than this being deprecated this option was never meant to be public. I'm glad we're doing the safe thing and marking it before removing it but can we also add (loud) warnings about using this? 

If passed to a normal operation this will leave the socket in a nondeterminate state and it will be put back into the pool with an unread reply. I would argue removing this option is a bug fix because "if set this option causes bugs", but we can at least document it.

Reply:
Resolved on slack, will be done in a separate ticket NODE-6377.